### ISSUE TITLE
Add Poisson multinomial loss

### ISSUE THREAD

**Issue by toby-clark4** [toby-clark4/borzoi-pytorch#1]
Check this repo and Chin's thesis implementation to see which is most integrable with my model.

- [ ] Loss function implementation chosen
- [ ] Add a check for single target prediction to enable switching between MSE for single targets vs multinomial for the full track prediction.
